db/fixtures/miq_policy_sets.yml

```diff
@@ -70,8 +70,8 @@
           action_type: default
           options: {}
       Condition:
-      - name: if container image has high severity openscap rule results
-        description: Has high severity OpenSCAP rule results
+      - name: if container image has no high severity failure in openscap rule results
+        description: No high severity failure in OpenSCAP rule results
```

Comment:
LGTM.
I tried to find history why test was opposite to reality but got lost in the double-negative you untangled in https://github.com/ManageIQ/manageiq/pull/16213/files#diff-1cc86ebb35ea7f32ab4c185735cd3612 ... :bowing_man: 
Once we had "annotate" action only on scan finding high severity issues; then we expanded to annotating both outcomes and I guess we forgot to update text then...